tests/conftest.py

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright 2018 Red Hat, Inc.
+#
+# This file is part of ansible-nmstate.
+#
+# ansible-nmstate is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ansible-nmstate is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with ansible-nmstate.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+import pytest
+
+
+@pytest.fixture
+def base_state():
```

Comment:
If you define the test data here it is somehow hidden from the reader, I would leave it in the test file for readability.

Reply:
I need it in two test files, IMHO having the same code twice would be not so nice as well.

Reply:
It's not code, but data. Up to you.

Reply:
It also needs to be a fixture for the other tests to work, so it will be code. I will keep it in mind and maybe reconsider in the future.